### DISCUSSION TITLE
Support printing escape codes

### DISCUSSION THREAD

**Discussion by miv391** [godotengine/godot-proposals#4123] [Scripting]
Then you could use colors, for example:

```gdscript
print('\x1b[1;31mThis is bold red text')
```

**Comment by Calinou** [comment]
See https://github.com/godotengine/godot/pull/33541.

**Comment by dalexeev** [comment]
There is [`print_rich()`](https://docs.godotengine.org/en/stable/classes/class_%40globalscope.html#class-globalscope-method-print-rich) (implemented by godotengine/godot#60675).

**Reply by Calinou** [reply]
For handling custom ANSI escape codes manually used in `print()` or `print_rich()`, https://github.com/godotengine/godot/pull/90900 and https://github.com/godotengine/godot/pull/92209 take care of it.
